app/src/main/java/com/sun/lol_info/ui/SplashActivity.kt

```diff
@@ -0,0 +1,21 @@
+package com.sun.lol_info.ui
+
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import androidx.appcompat.app.AppCompatActivity
+
+
+private const val TIME_WAITING = 2000L
+
+class SplashActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        Handler(Looper.getMainLooper()).postDelayed({
+            startActivity(MainActivity.getIntent(this))
+            finish()
+        }, TIME_WAITING)
```

Comment:
ko nên delay splash , nhất là với time 2000ms quá nhiều , splash chỉ dùng để khởi tạo ứng dụng chứ ko phải để show nội dung nên e ko cần cho user thẩm thấu cái hình splash của e đâu :))

Reply:
@vietnh-1109 vậy thì không cần delay hả anh ?

Reply:
ừa

Reply:
@vietnh-1109 em để delay 1s để sau này nó load data từ server về

Reply:
thế thì k cần delay , chắc gì load data đã <1s , đợi lúc load data xong r start activity cũng được mà

Reply:
Em build target sdk là 31 dành cho Android 12
Từ Android 12 người ta giới thiệu SplashScreen theo cách khác 
https://developer.android.com/about/versions/12/features/splash-screen#suspend-drawing
em có thể triển khai ở 1 pull riêng 

còn trước đó thì em có thể tham khảo ở 
https://shishirthedev.medium.com/the-right-way-to-implement-a-splash-screen-in-android-acae0e52949a
và không hề có delay gì cả, nếu cần load dữ liệu thì em load trong presenter bình thường, khi nào gọi xong gọi View để navigate tới Main Screen